### PR TITLE
Updated to NZBGet 15.0~r1196

### PR DESCRIPTION
**Pull request by cxhamilton**
Changes between NZBGet 14.1 and 15.0 may be found [here](http://nzbget.net/forum/viewtopic.php?f=4&t=1760#p11764).

### CODE REVIEW

File: spk/nzbget-testing/Makefile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 SPK_NAME = nzbget-testing
-SPK_VERS = 14.0~r1121
-SPK_REV = 6
+SPK_VERS = 15.0~r1196
+SPK_REV = 1
```
Comment (Dr-Bean) [inline spk/nzbget-testing/Makefile]: Make that a `7`, otherwise Package Center won't recognize it as an update.

**Comment by cxhamilton** [comment]
Done, though it is clear that I need to understand more about how Package Center behaves. I would think that the combination of SPK_VERS + SPK_REV would be significant. So, if SPK_VERS was updated, SPK_REV would naturally revert to 1, since it is the first revision of SPK_VERS.

**Comment by Dr-Bean** [comment]
Not difficult to understand: Package Center does not understand that `15.0~r1196` is a version number, so we add a version number with `SPK_REV`, which Package Center does understand. As a result, `SPK_REV` has to go up with every new release.

If you don't mind, please squash the two commits, that makes it cleaner to merge. I'll probably be able to merge/publish sometime in the next couple days, or maybe someone else will.

**Comment by cxhamilton** [comment]
Done and thanks for the clarification!

**Comment by Dr-Bean** [comment]
I'm seeing four commits now :P But I guess we can cherry-pick the right one :)

**Comment by cxhamilton** [comment]
Sorry about that. I have not squashed commits before.

**Comment by Shmp** [comment]
Is this update pushed to the synocommunity server? Cause i'm using the testing branch of nzbget but that is still stuck on v14.

**Comment by cxhamilton** [comment]
@Shmp No. The pull request has not yet been accepted.

**Comment by TeaPot91** [comment]
How can I get this onto my Synology NAS?

**Comment by Dr-Bean** [comment]
Thanks for the work though. I updated NZBGet-Testing to 15.0-1228 with 526e8bdbd2061292c2ee7963d6bde9f8c1dd6ac7, so let's close this PR. Note: Packages will be published over the next couple days, not sure when I get to it.

**Comment by Dr-Bean** [comment]
Nzbget-testing 15.0~r1228-7 is published on the repository.

**Comment by Shmp** [comment]
awesome!

**Pull request closed without merging**